pages/common/yacas.md

```diff
@@ -0,0 +1,28 @@
+# yacas
+
+> Yet Another Computer Algebra System.
+> More information: <http://www.yacas.org>.
+
+- Start an interactive console session:
+
+`yacas`
+
+- In the console session, execute a statement:
+
+`In> {{Integrate(x)Cos(x);}}`
+
+- In the console session, display an example:
```

Comment:
What kind of example is this? And is it random or something specific?

Reply:
The sample above, "Integrate(x)Cos(x);" serves to show the flavor of symbolic math statements accepted by the repl. This particular example computes the integral of the Cosine function and prints "Out> -Sin(x)". I could have used "{{statement}};" but a working example gives the user a better idea of how to use it. Another, simpler example might be "2+2;" but that might give the impression that yacas is a calculator, not a symbolic computer algebra system (like "Mathematica" and "sympy").

The yacas Example() function prints from a list of other example statements that you could try. The first one is "40!" or 40-factorial. There are many. They serve the same function as tldr, so I mentioned them. Both the Example() function and "quit" appear in an introduction paragraph when yacas starts an interactive session.